client/src/components/ratingsReviews/ReviewTile.jsx

```diff
@@ -3,60 +3,73 @@ import Helpfulness from '../shared/Helpfulness.jsx';
+                <StarRating rating={review.rating} max={5} />
+              </span>
+              <span id='rr-review-date' className='rr-review-date'>{review.reviewer_name}, {new Date(review.date).toLocaleDateString('en-EN', options)}</span>
+            </div>
             <br></br>
-            {review.photos.length > 0 ?
-              <div id='rr-review-photos' className='rr-review-photos'>
-                <DisplayPhotos photos={review.photos} />
-              </div> : null}
-            {review.recommend ?
-              <div id='rr-review-recommend' className='rr-review-recommend'>&#10003; I recommend this product!</div>
-              : null
-            }
-            {review.response ?
-              <div>
-                <br></br>
-                <div id='rr-review-response' className='rr-review-response'>
-                  <div id='rr-review-response title' className='rr-review-response title'>Response from seller</div>
-                  <div id='rr-review-response text' className='rr-review-response text'>{review.response}</div>
+            <div id='rr-review' className='rr-review'>
+              <div id='rr-review-summary' className='rr-review-summary'>{review.summary}</div>
+              <br></br>
+              <ReviewBody reviews={reviews} review={review}/>
+              <br></br>
+              {review.photos.length > 0 ?
+                <div id='rr-review-photos' className='rr-review-photos'>
+                  <DisplayPhotos photos={review.photos} />
+                </div> : null}
+              {review.recommend ?
+                <div id='rr-review-recommend' className='rr-review-recommend'>&#10003; I recommend this product!</div>
+                : null
+              }
+              {review.response ?
+                <div>
+                  <br></br>
+                  <div id='rr-review-response' className='rr-review-response'>
+                    <div id='rr-review-response title' className='rr-review-response title'>Response from seller</div>
+                    <div id='rr-review-response text' className='rr-review-response text'>{review.response}</div>
+                  </div>
                 </div>
+                : null
+              }
+              <br></br>
+              <div id='rr-review-feedback' className='rr-review-feedback'>
+                <Helpfulness review={review} />
+                <span> | </span>
+                <Report reviewId={review.review_id} />
+                <hr></hr>
               </div>
-              : null
-            }
-            <br></br>
-            <div id='rr-review-feedback' className='rr-review-feedback'>
-              <Helpfulness review={review} />
-              <span> | </span>
-              <Report reviewId={review.review_id} />
-              <hr></hr>
             </div>
           </div>
-        </div>
-      ))}
-    </div>
-  )
-    : (null);
-};
+        ))}
+      </div>
+    )
+      : (null);
```

Comment:
I'm not familiar enough with your component to know whether a placeholder here would make sense, but a single element here with some sort of message could let the user know that something is happening or supposed to happen before any content shows up. If it were roughly the size and shape that it would be with content, so much the better.

Reply:
Yes, will update